src/Form/Type/UiElement/VideoType.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Monsieurbiz\SyliusCmsPlugin\Form\Type\UiElement;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class VideoType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('video', FileType::class, [
+                'label' => 'monsieurbiz_cmsplugin.ui_element.video.field.video',
+                'data_class' => null, // @TODO check to have the File original data class, and remove this line
+                'required' => true,
+                'constraints' => [
+                    new Assert\NotBlank(),
+                    new Assert\File([
+                        'mimeTypes' => [
+                            'video/mp4',
+                            'video/mp3'
+                        ]
+                    ])
+                ],
+            ])
+            ->add('image', FileType::class, [
+                'label' => 'monsieurbiz_cmsplugin.ui_element.video.field.image',
+                'data_class' => null, // @TODO check to have the File original data class, and remove this line
```

Comment:
Why do you still have this TODO?

Reply:
I have an error if I remove the `data_class => null` option. 
```
The form's view data is expected to be an instance of class Symfony\Component\HttpFoundation\File\File, but is a(n) string. You can avoid this error by setting the "data_class" option to null or by adding a view transformer that transforms a(n) string to an instance of Symfony\Component\HttpFoundation\File\File
```

This is due to the fact that the field expect a file instance. When uploading there is no problem, but when updating, I only get the file path from the database so it's a string and not a file instance. 
I looked for a solution but I didn't found any. The only thing we can do is to define `data_class` option to `null`. See this [Stackoverflow comment ](https://stackoverflow.com/a/54633281/11398936)